v23.1/execution-locality-backups.md

```diff
@@ -0,0 +1,100 @@
+- If the leaseholder does not match the locality filter, the coordinator will select a node from those that match the locality filter with a preference for nodes with localities that are more similar to the leaseholder.
+
+When the coordinator assigns row data to a node matching the locality filter to back up, that node will read from the closest [replica](architecture/reads-and-writes-overview.html#architecture-replica). If the node is the leaseholder, or is itself a replica, it can read from itself. In the scenario where no replicas are available in the region of the assigned node, it may then read from a replica in a different region. As a result, you may want to consider [placing replicas](configure-replication-zones.html), including potentially non-voting replicas that will have less impact on read latency, in the locality or region you plan on pinning for backup job execution.
+
+For an overview of the backup job phases, see the [Backup Architecture](backup-architecture.html) page.
+
+## Examples
+
+This section outlines some possible use cases for the `EXECUTION LOCALITY` option.
+
+### Access backup storage restricted by network rules
+
+In certain cases, a cluster could be restricted by network rules across regions (or another locality requirement) for security purposes. This means that only nodes within the same region as the backup [cloud storage](use-cloud-storage.html) location can access the storage bucket. With `EXECUTION LOCALITY`, you can set up the backup to pin execution of the job to only the nodes with access to the bucket.
+
+The following diagram shows a CockroachDB cluster where each of the nodes can communicate with each other through a specific port, but any other network traffic between regions is blocked. Leaseholders in regions that do not match the cloud storage region (Node 2) cannot access the storage to [export backup data](backup-architecture.html#export-phase). 
+
+Instead, Node 3's locality matches a backup job created with `EXECUTION LOCALITY`. Leaseholders and replicas within that locality designation holding the row data to back up will begin reading and exporting to cloud storage.
+
+<img src="{{ 'images/v23.1/network-restriction.png' | relative_url }}" alt="Using execution locality when there is a network restriction between locality requirements" style="border:0px solid #eee;max-width:100%" />
+
+To execute the backup from only the region with access to cloud storage, you can specify [locality filters](cockroach-start.html#locality) that a node must match to take part in the backup job's execution.
+
+For example, you can pin the execution of the backup job to `us-west-1`: 
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+BACKUP DATABASE {database} INTO 'external://backup_storage_uswest' WITH EXECUTION LOCALITY = 'region=us-west-1', DETACHED;
```

Comment:
Is the comma before DETACHED correct? It looks like it should be `WITH DETACHED`: https://www.cockroachlabs.com/docs/stable/backup.html#run-a-backup-asynchronously

Reply:
Yes! This is correct, when you want to add multiple options, you would include commas between them. It looks a bit weird I think here because the k-v pairs kind of obscure the list of options. But, I tested and this works!